### PR TITLE
Support base64 encoding

### PR DESCRIPTION
**Pull request by maggarwal13**
Thunderbird 78+ is using OpenPGP to sign mail messages, rather than the Enigmail plugin.

The default for signed messages is now to base64 encode the message content, which we need to support.

### CODE REVIEW

File: whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java [hunk 1]
```diff
@@ -55,6 +56,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(MessageParser.class);
 
     private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("EEE MMM dd HH:mm:ss z yyyy");
+    public static final String HEADER_BASE_64 = "Content-Transfer-Encoding: base64";
```
Comment (eshryane) [inline whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java:59]: This is both the header and the value, maybe define each separately (also it's not guaranteed there will only be one space separating them).

Comment (maggarwal13) [reply whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java]: Changed it to use pattern matching.

Comment (eshryane) [reply whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java]: 👍

File: whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java [hunk 2]
```diff
@@ -328,4 +353,13 @@ void addCredential(final Credential credential) {
             credentials.add(credential);
         }
     }
+
+    private String decodeBase64Encoding(final String paragraph) {
+        try {
+            return new String(new Base64().decode(paragraph.getBytes()));
+        } catch (Exception e) {
+            LOGGER.info("failed to decode base64 for ");
```
Comment (eshryane) [inline whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java:361]: Missing ```e.getClass().getName()``` and ```e.getMessage()``` from LOGGER statement?

**Review by eshryane — COMMENTED** [review]
commented

File: whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java (hunk 2 above)
Comment (maggarwal13) [reply whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java]: Yes thanks for pointing it out, I somehow missed it

**Review by eshryane — APPROVED** [review]

**Pull request merged**